client/src/components/OnePlace/index.js

```diff
@@ -1,13 +1,27 @@
 import React, { Component } from 'react'
-import ListOfItems from '../ListOfItems'
+import './style.css'
 
 class OnePlace extends Component {
   render() {
-    //  this.props.match.url.split(":")[1])
-    const x = this.props.listOfItems
+    const { place } = this.props
+
     return (
       <div>
-        <p>one place item</p>
+        <div>
+          <img src={`${place.image1}`} />
+        </div>
+        <div className="placeInformation">
+          <h2 className="placeName">{place.name}</h2>
+
+          <h2 className="placeAvailibility">{place.availability}</h2>
+          <h2 className="placePrice">{place.price}</h2>
+          <p className="placeDescription">{place.description}</p>
```

Comment:
Did you see the place comment? I would either delete these from your classnames or use an hyphen 'place-price'.